vyper/compiler/phases.py

```diff
@@ -201,7 +211,9 @@ def generate_global_context(
     return GlobalContext.get_global_context(vyper_module, interface_codes=interface_codes)
 
 
-def generate_lll_nodes(global_ctx: GlobalContext) -> Tuple[parser.LLLnode, parser.LLLnode]:
+def generate_lll_nodes(
+    global_ctx: GlobalContext, use_ovm: bool
+) -> Tuple[parser.LLLnode, parser.LLLnode]:
     """
     Generate the intermediate representation (LLL) from the contextualized AST.
```

Comment:
Do we document the new arg in the comments?

Reply:
thanks, fixed